src/targets/maven.ts

```diff
@@ -411,4 +412,14 @@ export class MavenTarget extends BaseTarget {
         )
       : `${moduleName}.jar`;
   }
+
+  private logCmdOutput(output: Buffer | undefined): void {
```

Comment:
You shouldn't need any of this as `spawnProcess` does it underneath:

https://github.com/getsentry/craft/blob/a015c79de78bbc672570a38cb2177786e2fa93db/src/utils/system.ts#L174-L182